src/client/src/orangehrmPerformancePlugin/pages/ReviewSearch.vue

```diff
@@ -207,12 +208,14 @@ export default {
         const reviewer = item.reviewer?.employee;
         return {
           id: item.id,
-          employee: `${employee?.firstName} ${employee?.lastName} ${
-            employee?.terminationId ? ` ${$t('general.past_employee')}` : ''
-          }`,
-          reviewer: `${reviewer?.firstName} ${reviewer?.lastName} ${
-            reviewer?.terminationId ? ` ${$t('general.past_employee')}` : ''
-          }`,
+          employee: $tEmpName(employee, {
+            includeMiddle: false,
+            excludePastEmpTag: false,
+          }),
```

Comment:
```suggestion
          employee: $tEmpName(employee),
```
@devishke-orange both options are false by default right?

Reply:
yes no need to have the options object in this case